build/localnet/cluster-manager/utils.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/pokt-network/pocket/shared/crypto"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func fetchValidatorPrivateKeys(clientset *kubernetes.Clientset, validatorKeysMap map[string]crypto.Ed25519PrivateKey) {
+	private_keys_secret, err := clientset.CoreV1().Secrets("default").Get(context.TODO(), "v1-localnet-validators-private-keys", metav1.GetOptions{})
+	if err != nil {
+		panic(err)
+	}
+
+	for key, value := range private_keys_secret.Data {
+		pk, err := parsePrivateKey(bytes.NewReader(value))
+		if err != nil {
+			panic(err)
+		}
+		validatorKeysMap[key] = pk
+	}
+}
+
+func isValidator(service *v1.Service) bool {
+	return service.Labels["v1-purpose"] == "validator"
+}
+
+func extractValidatorId(validatorName string) string {
+	if len(validatorName) >= 3 {
+		return validatorName[len(validatorName)-3:]
+	}
+	return validatorName
+}
+
+func parsePrivateKey(reader io.Reader) (pk crypto.Ed25519PrivateKey, err error) {
```

Comment:
We have a `crypto.PrivateKey` interface. Any chance we could use that instead?

We're not great at it (yet), but I do want to try avoiding using the direct types throughout the codebase if possible when a more general purpose interface is available